### ISSUE TITLE
Following the Keycloak OpenJDK guide leads me to "Bad request: Invalid parameter: redirect_uri"

### ISSUE THREAD

**Issue by tommai78101** [keycloak/keycloak#13761]
### Describe the bug

I get an error message while following the set up guide here:

https://www.keycloak.org/getting-started/getting-started-zip

![image](https://user-images.githubusercontent.com/6444116/184426656-7de03142-3ae7-4c7f-beae-6b7ed51db2b3.png)

Steps to reproduce:

### Specs:

Keycloak 19.0.1 OpenJDK
Java 17 LTS
Ubuntu 20.04-LTS on Windows 10 WSL2

### Version

19.0.1

### Expected behavior

I should be able to continue following the setup guide.

### Actual behavior

It returns an error page , BAD REQUEST.  No other errors were found nor given.

![image](https://user-images.githubusercontent.com/6444116/184428293-d1c31840-e9a1-4109-bbae-4de5c45826bd.png)

### How to Reproduce?

1. Follow the guide from the beginning, all the way to "Create an admin user".
2. For the admin user, fill in the blanks with the following:
    * Username: `admin@example.com`
         * Must use an email address as the username. 
    * Password: `something1`
3. Continue with the rest of the guide up to "Create a realm".
4. When creating the realm, use the realm name, `Test Realm`.
      * Must have a whitespace in it.
5. Continue following the guide up to "Login to account console".
6. Go to the account console using the following URL:    `http://localhost:8080/realms/Test%20Realm/account`
7. You should see the Test Realm bad request page, shown above.

### Anything else?

If there is anything I'm doing wrong, please let me know.

**Comment by DGuhr** [comment]
Hi @tommai78101 whitespaces in realms are not easily possible and discouraged for future usage. See the discussion here: https://github.com/keycloak/keycloak/discussions/11978 
If you want to make it work by all means, see e.g. @mposolda  answer in the discussion: https://github.com/keycloak/keycloak/discussions/11978#discussioncomment-2773583 (using a wildcard `*` for valid redirect uri), but this has other implications on security. 

Long story short: Pls don't use whitespaces in realm names, we'll do sth about the ability to create realms with whitespaces in the future, but as it was possible some time in the past the change is not as easy. If you really want to, there are ways to circumvent this restriction for now.

Closing this ticket, feel free to join the discussion if you have a valid use-case for necessary whitespaces :)

**Comment by tommai78101** [comment]
Thank you for letting me know.
